app/build.gradle

```diff
@@ -24,7 +24,7 @@ dependencies {
     compile 'com.android.support:appcompat-v7:25.3.1'
     compile 'com.android.support:recyclerview-v7:25.3.1'
     compile 'com.android.support:design:25.3.1'
-    compile 'com.android.support:preference-v7:25.3.1'
+    implementation 'com.android.support:preference-v7:25.3.1'
```

Comment:
this is not good, since the build tools are not 3.0.0
remove this as it was before with compile